Api/Endpoints/Files.php

```diff
@@ -68,6 +68,9 @@ public function fetchAll($projectId)
      */
     public function upload($projectId, $body)
     {
+        // Requests that do not use the queue=true parameter are deprecated and will be unsupported
+        $body['queue'] = true;
```

Comment:
Just double-check - is it really goes to body? I thought, it was query parameter

Reply:
Ye, upload doesn't expect any specific query params.
https://app.lokalise.com/api2docs/curl/#resource-files